src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,26 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
 
+    public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = (ManufacturerDao) injector
+                .getInstance(ManufacturerDao.class);
+        Manufacturer volkswagen = new Manufacturer("Volkswagen", "Germany");
+        Manufacturer kia = new Manufacturer("KIA", "South Korea");
+        volkswagen = manufacturerDao.create(volkswagen);
+        kia = manufacturerDao.create(kia);
```

Comment:
```suggestion
        manufacturerDao.create(volkswagen);
        manufacturerDao.create(kia);
```

no need to reassign variable